src/controllers/expensesControllers.js

```diff
@@ -0,0 +1,83 @@
+const {
+    getAllExpenses,
+    getExpenseById,
+    addExpense,
+    updateExpense,
+    deleteExpense,
+  } = require('../services/expensesService');
+  
+  export const get = (req, res) => {
+    res.send(getAllExpenses);
+  };
+  
+  export const getOne = (req, res) => {
+    const { id } = req.params;
+  
+    const expense = getExpenseById(id);
+  
+    if (!expense) {
+      res.statusCode(404);
+      res.end('Expense not found');
+      return;
+    }
+  
+    res.send(expense);
+  };
+  
+  export const add = (req, res) => {
+    const { userId, spentAt, title, amount, category, note }  = req.body;
+  
+    if (!userId || !spentAt || !title || !amount || !category) {
+      res.statusCode(404);
+      res.end('Name is required');

```

Comment:
and if it's some other field that's not being passed?
it will be confusing for a user